### ISSUE TITLE
[Solution] Adding SSO Authentication to the login screen

### ISSUE THREAD

**Issue by tenekev** [9p4/jellyfin-plugin-sso#120]
Hello, I just set up the plugin and it works nicely. 

I wanted a SSO Login button on the login screen and realized that Jellyfin's "Login Disclaimer" doesn't sanitize the input. 

This means you can add HTML content into the _Disclaimer field_ and it will render it in the UI. This is a **very cheap** way to add functionality and in my opinion, it's indistinguishable from a custom solution.

### Here is my solution.

```html
<a href="/sso/OID/p/Authelia" style="padding: 1rem; width: 350px; border-radius: 10vh; display: block;">
    SSO Login
</a>
```

### I added it in Dashboard > General > Branding > Login disclaimer
![image](https://github.com/9p4/jellyfin-plugin-sso/assets/30023563/e312ae08-39cd-444d-9288-9a40feb83092)

### Here is the result
![image](https://github.com/9p4/jellyfin-plugin-sso/assets/30023563/d31f7290-f6f5-4419-b9ff-40141c9c2a40)

### Here is with an icon
```html
<a href="/sso/OID/p/Authelia" style="padding: 1rem; width: 350px; border-radius: 10vh; display: block;">
  <span class="material-icons input" style="margin-right:1rem"></span>
  SSO Login
</a>
```
I hope this can be added to the explanation. I'm going to make a full tutorial for Jellyfin + SSO + LLDAP + Authelia OIDC but I feel that this particular nugget is applicable for everyone.

**Comment by 9p4** [comment]
See also https://github.com/9p4/jellyfin-plugin-sso/issues/16

**Comment by 9p4** [comment]
Fixed in 076cb1b893711b664386d46f6ee8d776243a9061
